docs/tutorial.rst

```diff
@@ -213,6 +216,9 @@ supports the Common Lisp method of quoting:
    => '(1 2 3)
    (1L 2L 3L)
 
+Note the output is in Python 2. In Python 3 it is `(1 2 3)`
+because Python 3 does not support `Long`.
```

Comment:
This is kind of confusing. Maybe, in the earlier sections of the tutorial, it should mention the int <> long difference and how Hy handles it, and then, down here, just refer to that section.